rskj-core/src/main/java/co/rsk/rpc/netty/RskJsonRpcHandler.java

```diff
@@ -63,10 +63,8 @@ public RskJsonRpcHandler(EthSubscriptionNotificationEmitter emitter, JsonRpcSeri
 
     @Override
     protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder msg) {
-        try {
-            RskJsonRpcRequest request = serializer.deserializeRequest(
-                    new ByteBufInputStream(msg.copy().content())
-            );
+        try (ByteBufInputStream bis = new ByteBufInputStream(msg.copy().content())) {
```

Comment:
Well, this class is only used for websocket server. It produces memory leaks when the server is shut down due to a write timeout. 

More precisely this part

```java
msg.copy().content()
```

Reply:
Perfect!! Now it is ready!

Reply:
nice!